web/server/codechecker_server/cmd/server.py

```diff
@@ -559,51 +562,95 @@ def __init__(self, product_name):
         self.product_name = product_name
 
 
-def __db_migration(cfg_sql_server, migration_root, environ,
-                   product_to_upgrade: str = 'all',
-                   force_upgrade: bool = False):
+def __db_migration(migration_root,
+                   environ,
+                   endpoint: str,
+                   connection_string: str,
+                   init_instead_of_upgrade: bool) -> DBStatus:
+    try:
+        db = database.SQLServer.from_connection_string(connection_string,
+                                                       endpoint,
+                                                       RUN_META,
+                                                       migration_root,
+                                                       interactive=False,
+                                                       env=environ)
+        if init_instead_of_upgrade:
+            LOG.info("[%s] Initialising...", endpoint)
```

Comment:
I think we are pretty firmly embedded in US english, certainly for "initialization".